test/libsolidity/solidityExecutionFramework.h

```diff
@@ -40,23 +40,27 @@ namespace test
 class ExecutionFramework
 {
 public:
-	ExecutionFramework() { g_logVerbosity = 0; }
+ExecutionFramework() { g_logVerbosity = 0; }
 
-	bytes const& compileAndRun(std::string const& _sourceCode, u256 const& _value = 0, std::string const& _contractName = "")
+	bytes const& execute(std::string const& _sourceCode, u256 const& _value = 0, std::string const& _contractName = "")
```

Comment:
do you have a good name for this function? it does the same as compile and run only doesn't check the output